docs/Future_vision.md

```diff
@@ -0,0 +1,43 @@
+# Future Vision
+
+## Where to go from here?
+
+Liquid still hasn't reached its full potential. We wanted to do a lot more than we managed to do in this semester. You can look through our user stories on github or zenhub (please beware that the user stories project on github is outdated, because we switched to zenhub). To present a clear overview here is a short list (please refer to the [contributing doc](https://github.com/hpi-swa-teaching/Liquid/blob/main/CONTRIBUTING.md) for term definitions):
```

Comment:
```suggestion
Liquid still hasn't reached its full potential. We wanted to do a lot more than we managed to do in this semester. You can look through our user stories on github or zenhub (please beware that the user stories project on github is outdated, because we switched to zenhub). Following is a brief overview of possible improvements. Terminology is explained in our [term definitions](https://github.com/hpi-swa-teaching/Liquid/blob/main/CONTRIBUTING.md#term-definitions).
```